### PR TITLE
Deprecated usage of The Events Calendar old Asset Loading methods

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#40267](http://central.tri.be/issues/40267)
---

It's still super unsafe to any Notices for Deprecation of file here yet.

### CODE REVIEW

**Comment by ggwicz** [comment]
Removing `merge` tag for reasons noted in [this Slack message](https://tribe.slack.com/archives/C0258NY1C/p1522692331000345).

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *M18.07*, but this was pull requested against *release/M18.06*.

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *M18.08*, but this was pull requested against *release/M18.07*.

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *M18.09*, but this was pull requested against *release/M18.08*.

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *M18.11*, but this was pull requested against *release/M18.10*.

File: readme.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 = [4.7.18] TBD =
 * Fix - Add `target="_blank"` to repository links in the Help Page [107974]
+* Tweak - Deprecate the usage of old asset loadeing methods [40267]
```
Comment (mitogh) [inline readme.txt]: Typo `loadeing`

Comment (juanfra) [reply readme.txt]: great catch, thanks @mitogh

**Comment by mitogh** [comment]
Conflicts @juanfra !

**Pull request merged**